### PR TITLE
Walled gardens design

### PR DESCRIPTION
**Pull request by gericci**
## Summary of Changes
- Walled Gardens page structure
- "cleaned" URL in links
- First design touches

### CODE REVIEW

**Comment by ssddanbrown** [comment]
Thought I'd copy [the feedback](https://discord.com/channels/945208532930822155/948084147837075477/959937780367114281) from @mtom55 on Discord into an actionable list here. Please see my note afterwards in reference to the first and last item.

- [ ] Table of contents - Could we get the list of all headings down to h3 h4 here along with the heading numbers.  It’s really useful if you want to quickly jump to a particular argument or if someone is browsing for specific section - [Reference Image](https://user-images.githubusercontent.com/8343178/161426122-7efd3e99-11e0-4fc9-9051-e7bbc6c95545.png)
- [ ] Headings - I think although the lowercase headings look good, we probably need to keep the original case to make it look more formal - [Reference Image](https://user-images.githubusercontent.com/8343178/161426152-a0035192-76e1-4200-9804-42c86ba676ad.png)
- [ ] Mobile Scroll - I think this link is causing the horizontal scroll - [Reference Image](https://user-images.githubusercontent.com/8343178/161426196-5590833a-43b8-49ca-9a02-ad5af1809be7.png)
- [ ] Headings - Can we make margin above the h4s bigger than below - [Reference Image](https://user-images.githubusercontent.com/8343178/161426218-c22d4a99-7a83-42b6-b5e0-ad122e991f85.png)
- [ ] Images in lists - Not quite sure what do here.  Maybe just have title image. - [Reference Image A](https://user-images.githubusercontent.com/8343178/161426261-dd829226-4fdd-4bd9-b9bf-c668918afc69.png) - [Reference Image B](https://user-images.githubusercontent.com/8343178/161426265-6d9e8567-e7c7-41d1-9639-a6252e6a1b2c.png)
- [ ] Banner Example Images - This bit looks weird. Maybe we can stick headings in between?   Like. “1. install banner at top” - [Reference Image](https://user-images.githubusercontent.com/8343178/161426307-4762397d-9a10-41e9-a9f3-833e3cf58cae.png)
- [ ] Section Links - Last one is there anyway to allow users to easily copy a link that directly takes them to a section - [Reference Image](https://user-images.githubusercontent.com/8343178/161426152-a0035192-76e1-4200-9804-42c86ba676ad.png)

#### Table of Contents & Headings

In reference to the first and last items in the list above, it might be worth leaving these to instead focus on all other design/layout/style based changes, for the scope of this specific PR.
I've opened PR #19 to auto-add ids and anchor links to all markdown headers to save doing this manually.
I think we should be able to auto-generate the table-of-contents here also based on the headings. Just mentioning here so you don't waste any time doing manually, if we can automatically do this later, especially if the heading anchor links may change due to #19. 
Therefore I'd advise this PR ignores headings & table-of-contents for now (Apart from the design work already done) then we address that after both this PR, and PR #19, have been merged.

File: src/css/main.css [hunk 1]
```diff
@@ -572,3 +585,31 @@ dl.glossary {display: block;}
...
+	margin-bottom: 0.5em;
+}
+
+.p-author a {font-size: 80%;}
+
+div.toc ul {margin-left: 2.5rem;}
+div.toc li {font-size: 0.9em;}
+
+p.download {
+	background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24' width='24' height='24'%3E%3Cpath fill='none' d='M0 0h24v24H0z'/%3E%3Cpath d='M16 2l5 5v14.008a.993.993 0 0 1-.993.992H3.993A1 1 0 0 1 3 21.008V2.992C3 2.444 3.445 2 3.993 2H16zm-3 10V8h-2v4H8l4 4 4-4h-3z' style='fill:%23CD4C1D' /%3E%3C/svg%3E");
```
Comment (sbesh91) [inline src/css/main.css:608]: If this image might be reused, would it be possible to tuck it into a css variable?  A named variable would give some context to what we're loading as well.

Comment (gericci) [reply src/css/main.css:607]: Yes, no problem. Not sure it will be used a lot though (or modified)

**Comment by gericci** [comment]
@ssddanbrown  Done issues 1 to 5

**Comment by aliblackwell** [comment]
This is looking absolutely fantastic. Cracking work @gericci, @ssddanbrown & @thescientist13!

🚀

**Pull request merged**